cypress/integration/brewery/FileParameters-tests.spec.js

```diff
@@ -117,10 +117,8 @@ describe('Simple operations on a file parameter', () => {
 
   it('can delete an uploaded file and run the scenario', () => {
     // Wait required to prevent some kind of race condition somehow
-    cy.wait(5000); // eslint-disable-line cypress/no-unnecessary-waiting
-    Scenarios.selectScenario(firstScenarioName, firstScenarioId);
-    cy.wait(5000); // eslint-disable-line cypress/no-unnecessary-waiting
-    ScenarioParameters.edit(120);
+    Scenarios.selectScenario(firstScenarioName, firstScenarioId, 5);
```

Comment:
`5` could be the default value for the `selectTimeout` argument (c.f. my previous comment)

Reply:
Same as above, I put default timeout in getter function :
`function getScenarioSelector(timeout = 5) {
  return cy.get(GENERIC_SELECTORS.scenario.selectInput, { timeout: timeout * 1000 });
}`